frontend/src/interfaces.ts

```diff
@@ -60,6 +66,7 @@ export interface Sinks<State> {
     dataQuery?: Stream<{ type: string; id: string }>;
     commandGateway?: Stream<any>;
     map?: Stream<Command<MutateMapEventData[]>>;
+    building?: Stream<Command<BuildingEventData[]>>;
```

Comment:
I presumed that as there are sources in buildingDriver this is not needed, and it might confuse as is does nothing.

Reply:
removed